net-misc/netkit-timed/netkit-timed-0.17-r13.ebuild

```diff
@@ -9,17 +9,16 @@ DESCRIPTION="Netkit - timed: Time daemon"
 HOMEPAGE="https://wiki.linuxfoundation.org/networking/netkit"
 SRC_URI="http://ftp.linux.org.uk/pub/linux/Networking/netkit/${P}.tar.gz"
 
-KEYWORDS="amd64 ~mips ppc ppc64 sparc x86"
 LICENSE="BSD GPL-2"
+KEYWORDS="~amd64 ~x86"
```

Comment:
You are dropping the last remaining stable ebuild with this, and dropping other currently supported architechtures with it. I'd recommend just doing a new revision copy and **not** deleting the current -r11 ebuild.